src/injected/textSelectorEngine.ts

```diff
@@ -48,10 +48,27 @@ export function createTextSelector(shadow: boolean): SelectorEngine {
   return engine;
 }
 
+function unescape(s: string): string {
+  if (s.indexOf('\\') === -1)
```

Comment:
nit: I personally would use there .includes for readability

Reply:
Good point!